docker/Dockerfile

```diff
@@ -119,6 +121,17 @@ RUN \
   cd s3cmd-${S3_CMD_VERSION} && \
   python setup.py install
 
+# Install conda
+RUN \
+  curl -sSL https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -o /tmp/miniconda.sh && \
+  bash /tmp/miniconda.sh -bfp /usr/local && \
+  rm -rf /tmp/miniconda.sh && \
+  conda install -y anaconda-client conda-build && \
+  conda update conda && \
+  conda config --append channels conda-forge && \
```

Comment:
Is conda-forge only needed for tabulate or also for something else?

Reply:
tabulate only. Should I install tabulate using this channel rather than adding it globally?

Reply:
I think this is fine as long as the tests (of installing from conda) don't rely on this docker image or at least this conda env.

What I am worried about that the tests will pass fine because we have enabled some channel that the user doesn't have. The tests should emulate the user.

Reply:
I've removed the channel from root conda environment. It's added on demand by the release pipeline.